### DISCUSSION TITLE
Next RQ release?

### DISCUSSION THREAD

**Discussion by hberntsen** [rq/rq#2115] [General]
Several improvements have made it to the master branch and I'd like to use them. What's the status w.r.t. releasing a new RQ version? I also see some references to v2.0 here and there.

**Comment by craigds** [comment]
I'd also like to know. I'm looking to update, but I'm pretty confused about the 1.16.x releases.

It looks like they've been made from the `v1` branch, which is missing some things from master.

e.g. my fix from #1970 (Sept 2023) still isn't in the `v1` branch, so there's no release that contains it yet.

Also oddly, master has `1.16.2` in the version.py file (because `v1` has been merged into master, but not the other way around)

**Comment by anton-daneyko-ultramarin** [comment]
I am also interested in the new release. If https://github.com/rq/rq/pull/2133 is merged, this is going to fix a problem in my project, otherwise I will be on a journey for some shady workaround.

@selwin Could you share your plan for upcoming releases?

**Reply by anton-daneyko-ultramarin** [reply]
@craigds How do you go by not having the fix you need in the meantime? I also need a feature that is missing in the 1.16.2 and considering my options like building my own package with custom version or pulling RQ as `git submodule`.

**Comment by selwin** [comment]
Hey really sorry about this, it has taken me much longer than expected to rework RQ’s internals to for the 2.0 release.

I’m currently getting Django-RQ to support version 2.0, it should be released at the end of this month. 

Sorry for the delay 🙏🏻

**Reply by selwin** [reply]
I’d also appreciate it if someone can take thr master branch for a spin and report any oddities.

**Reply by genericmoniker** [reply]
I see that there is a 2.0.0 on pypi.org now, which is great. Was that released from the master branch, or is the request to take master for a spin in anticipation of a different future release?

I was hoping issue #2044 would be addressed in the next release, but it appears to not be included even though there was a #2055 PR merged into master earlier this year. Sorry, I'm just confused a bit as to what to expect.

**Reply by selwin** [reply]
@genericmoniker this should already be fixed in the 2.0 release. I checked `utils.py` in https://pypi.org/project/rq/#files and `utcnow()` is no longer in there.

**Reply by genericmoniker** [reply]
Thank you for dispelling my confusion! I was checking the 2.0.0 version of one install of rq while accidentally running a different one that was older.
